spec/griddler/sendgrid/adapter_spec.rb

```diff
@@ -23,15 +23,15 @@
       attachment2: upload_2,
      'attachment-info' => <<-eojson
         {
-          'attachment2': {
-            'filename': 'photo2.jpg',
-            'name': 'photo2.jpg',
-            'type': 'image/jpeg'
+          "attachment2": {
+            "filename": "photo2.jpg",
+            "name": "photo2.jpg",
+            "type": "image/jpeg"
           },
-          'attachment1': {
-            'filename': 'photo1.jpg',
-            'name': 'photo1.jpg',
-            'type': 'image/jpeg'
+          "attachment1": {
+            "filename": "photo1.jpg",
+            "name": "photo1.jpg",
```

Comment:
@tabfugnic The quotes here aren't Ruby string literals, they're part of a heredoc that will be parsed by `JSON.parse`. Single quotes are invalid JSON, so these needed to be changed to double quotes when `JSON.parse` was introduced in the adapter. Previously the entire `'attachment-info'` key/value pair was being discarded, so the invalid JSON was not caught.